pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -1153,7 +1165,7 @@ fun PNConfiguration.toJs(): PubNubJs.PNConfiguration {
     config.subscribeKey = subscribeKey
     config.publishKey = publishKey
     config.secretKey = secretKey
-    config.logVerbosity = logVerbosity
+    config.logVerbosity = logVerbosity == PNLogVerbosity.BODY
```

Comment:
Do we want to have `PNLogVerbosity.BODY` as default ?

Reply:
please note that this checks for equality `==` and not `=`, so it's only set to true when `BODY` is passed in arguments